### PR TITLE
re #4 adding coerce to atom

### PR DESCRIPTION
**Pull request by lowks**
Implementation of the `coerce` for atom

### CODE REVIEW

File: lib/coercion.ex [hunk 1]
```diff
@@ -80,6 +85,19 @@ defmodule Coercion do
...
 
+  # Atom
+  def coerce(value, :atom) when is_bitstring(value) do
+    atom_value =
+      value
+      |> String.trim
+      |> String.to_existing_atom
+    {:ok, atom_value}
+  end
+  def coerce(value, :atom) when is_bitstring(value), do: coerce(value, :atom)
```
Comment (moxley) [inline lib/coercion.ex]: This signature looks like a duplicate of the one above.

File: lib/coercion.ex [hunk 2]
```diff
@@ -45,7 +45,12 @@ defmodule Coercion do
       {:ok, "true"}
       iex> coerce(10.5, :string)
       {:ok, "10.5"}
-
+      iex(3)> :bellow
```
Comment (moxley) [inline lib/coercion.ex]: Can you remove the prompt numbers from these examples?

**Review by moxley — CHANGES_REQUESTED** [review]
Looks good. Just a couple tweaks.

File: lib/coercion.ex (hunk 2 above)
Comment (lowks) [reply lib/coercion.ex]: Done.

File: lib/coercion.ex (hunk 1 above)
Comment (lowks) [reply lib/coercion.ex]: Done

**Comment by lowks** [comment]
@moxley the changes have been done. Please have a look.

File: lib/coercion.ex [hunk 3]
```diff
@@ -45,7 +45,12 @@ defmodule Coercion do
       {:ok, "true"}
       iex> coerce(10.5, :string)
       {:ok, "10.5"}
-
+      iex> :bellow
```
Comment (moxley) [inline lib/coercion.ex:48]: Looks good to merge, but did you mean "bellow" or "below", which are two completely different words. Just wanted to make sure this was what you intended.

**Review by moxley — CHANGES_REQUESTED** [review]

File: lib/coercion.ex (hunk 3 above)
Comment (lowks) [reply lib/coercion.ex:48]: `bellow` is accurate

**Review by moxley — APPROVED** [review]

**Comment by moxley** [comment]
Thanks @lowks!

**Pull request merged**